data/migrations/20190118132117_users.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('users', tbl => {
+
+    tbl.increments();
+
+    tbl.string('username', 20).unique().notNullable();
+    tbl.string('password').notNullable();
+
+  });
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropTableIfExists('users');
```

Comment:
As a thing of note, this is a super common function for the `down` in migrations. In projects where you have more than one migration, it'd definitely be worth it to make a helper function:
```javascript
const dropTable = tbl => knex => knex.schema.dropTableIfExists(tbl);

// and usage:
exports.down = dropTable('users');
```